Token.js

```diff
@@ -142,6 +143,7 @@ class Token {
 	}
 	move(top, left) {
 		if (this.options.locked) return; // don't allow moving if the token is locked
+		if (this.options.restrictPlayerMove) return; // don't allow moving if the token is locked
```

Comment:
same as before... you cannot just stop movement if the option is set..   you need to check if window.DM